docs/english/test/index.md

```diff
@@ -57,9 +30,9 @@ The ERP-system must do the following:
 
 Skatteetaten have made available a test application that shows how the steps described above can be done. This is written in [jupyter notbook format](https://jupyter.org/):
 
-1. [Jupyter notebook demo for retrieving, validation](https://github.com/Skatteetaten/mva-meldingen/blob/master/docs/documentation/test/demo.ipynb)Download the folder testinnsending and run the script demo.ipynb (the script will execute all steps in the process: call ping service to check the connection and validate the mva tax return.
+1. [Jupyter notebook demo for retrieving, validation](https://github.com/Skatteetaten/mva-meldingen/blob/master/docs/documentation/test/demo.ipynb)Download the folder test and run the script demo.ipynb (the script will execute all steps in the process: call ping service to check the connection and validate the mva tax return.
 2. [Jupyter notebook demo for retrieving, validation and submitting](https://github.com/Skatteetaten/mva-meldingen/blob/master/docs/documentation/test/innsending-eksempel.ipynb). Run the script demo.ipynb. The script will execute all steps in the process.
-3. [Pyton script to fetch token](https://github.com/Skatteetaten/mva-meldingen/blob/master/docs/documentation/test/Steg/logge_inn_idporten.py) and [postman script to validate mva tax return](https://github.com/Skatteetaten/mva-meldingen/blob/master/docs/documentation/test/MeldingValidering.postman_collection.json). First step is to log in to ID-porten (described later), and store the token in the format "Bearer <em>hentet-token</em>" as an environment variable by the name "test-bearer" in postman, then use the postman script in order to validate the mva vat return.
+3. [Pyton script to fetch token](https://github.com/Skatteetaten/mva-meldingen/blob/master/docs/documentation/test/Steg/logge_inn_idporten.py) and [postman script to validate mva tax return](https://github.com/Skatteetaten/mva-meldingen/blob/master/docs/documentation/test/MeldingValidering.postman_collection.json). First step is to log in to ID-porten (described later), and store the token in the format "Bearer <em>token-here</em>" as an environment variable by the name "test-bearer" in postman, then use the postman script in order to validate the mva vat return.
 4. [Example XML files](https://github.com/Skatteetaten/mva-meldingen/tree/master/docs/documentation/test/eksempler/melding)
 
 ## Start using ID-porten
```

Comment:
"When ordering access provide information that you would like to access the skattemeldings-API from Skatteetaten."
Skal det kanskje være noe komma?

"which in turn provides access to the administration of the **Customer's** use of"
Customer med liten _c_?